checker/src/org/checkerframework/checker/fenum/qual/FenumBottom.java

```diff
@@ -1,14 +1,18 @@
 package org.checkerframework.checker.fenum.qual;
 
-import java.lang.annotation.*;
-
-import com.sun.source.tree.Tree;
-
 import org.checkerframework.framework.qual.DefaultFor;
-import org.checkerframework.framework.qual.TypeUseLocation;
 import org.checkerframework.framework.qual.ImplicitFor;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+import java.lang.annotation.Documented;
```

Comment:
Minor: I would have expected to see java.* annotations at the beginning or end rather than in the middle, on the theory that import statements go from most general libraries to most specific libraries, but maybe we do something else?  It's not alphabetical, but I can't tell what the rule is.